### PR TITLE
Rubocop and other updates/fixes for TK

### PR DESCRIPTION
**Pull request by ramereth**

### CODE REVIEW

**Comment by ramereth** [comment]
@knightsamar that should be fixed now. Thanks for catching that!

File: test/integration/roles/openstack_tk.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "env_run_lists": {},
   "run_list": [
     "recipe[openstack_test::hosts]",
+    "recipe[openstack_test::cacert]",
```
Comment (knightsamar) [inline test/integration/roles/openstack_tk.json]: Where exactly is this cookbook openstack_test ?

Comment (knightsamar) [inline test/integration/roles/openstack_tk.json]: Test kitchen works well btw 👍

Comment (ramereth) [inline test/integration/roles/openstack_tk.json]: @knightsamar it's in the cookbook, look at Berksfile to see where exactly

Comment (knightsamar) [inline test/integration/roles/openstack_tk.json]: Thanks. This is an interesting setup. Hopefully I can make time to read the code and understand how you are making these things work.

**Comment by ramereth** [comment]
!bump patch *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v2.3.1.  Have a nice day!

**Pull request merged**